lib/commonOpts.ts

```diff
@@ -1,44 +1,55 @@
-// @ts-nocheck
-const {
+import {
   LOG_LEVEL,
-  setLogLevel: setLoggerLogLevel,
-} = require('@hubspot/local-dev-lib/logger');
-const {
-  DEFAULT_MODE,
-  MODE,
-} = require('@hubspot/local-dev-lib/constants/files');
-const {
-  getAccountId: getAccountIdFromConfig,
+  setLogLevel as setLoggerLogLevel,
+} from '@hubspot/local-dev-lib/logger';
+import { DEFAULT_MODE, MODE } from '@hubspot/local-dev-lib/constants/files';
+import {
+  getAccountId as getAccountIdFromConfig,
   getAccountConfig,
   getAndLoadConfigIfNeeded,
-} = require('@hubspot/local-dev-lib/config');
-const { i18n } = require('./lang');
+} from '@hubspot/local-dev-lib/config';
+import { i18n } from './lang';
+import { Argv, Arguments } from 'yargs';
+
+interface CustomOptions {
```

Comment:
I use this type twice here and in the `logDebugInfo` file. Should I create a Types file in the `lib` folder? Wanted to check, if we already had plans about it.

Reply:
I think you could even create a new `types` folder like we have in LDL at the top level. I'm not really sure what the best way to handle this specific type is though, since this theoretically represents anything that could be passed in via a flag. For right now I could see it making sense to have each function only specify what they need from the `Arguments` type, rather than sharing all of them. For example, in `getMode`, it could just take in `Arguments<{ mode?: string }>` or something like that. Feel like it'll become clearer how to handle this once we start converting commands

Reply:
Per our Huddle on Friday, I changed the code to have each function only specify what they need from the `Arguments` type. Going to resolve for now, but feel free to reopen.